lineid_plot/lineid_plot.py

```diff
@@ -8,8 +8,9 @@
 import numpy as np
 from matplotlib import pyplot as plt
 
-__version__ = "0.5"
-__author__ = "Prasanth Nair"
+__all__ = ['plot_line_ids', 'initial_annotate_kwargs', 'initial_plot_kwargs',
+           'unique_labels', 'get_line_flux', 'get_box_loc', 'adjust_boxes',
+           'prepare_axes']
```

Comment:
I put here all the methods that were public, but you may want to remove some ?

Reply:
The public API is just the function 'plot_line_ids'. So perhaps we can have just that function in "__all__"?

Reply:
Most of the other functions are also mentioned in the docs, that's why I put all the functions (that were importable from `lineid_plots`).

Reply:
Oh! I haven't read the documentation in a while! We will keep the list you have.